apps/aecore/test/aec_mining_tests.erl

```diff
@@ -41,7 +41,8 @@ mine_block_test_() ->
                  Nonce = case aec_hard_forks:protocol_effective_at_height(Height + 1) of
                              ?ROMA_PROTOCOL_VSN    -> 1566115190779737391;
                              ?MINERVA_PROTOCOL_VSN -> 9700108337194733977;
-                             ?FORTUNA_PROTOCOL_VSN -> 5415506607407149835
+                             ?FORTUNA_PROTOCOL_VSN -> 5415506607407149835;
+                             ?LIMA_PROTOCOL_VSN    -> 13759263286250281607
```

Comment:
Is this actually exercised?

Reply:
What I mean is: Does this test fail without this change?

Reply:
I did what the comment says in: https://github.com/aeternity/aeternity/pull/2495/files/1579c5f5aca7f8c26a8d5005d80fbb51d95f3cad..5f8269c159e44add588fe0d640ab3cd9699f9251#diff-d65f9ffa9357da34dce099a87063193fR35

Reply:
> What I mean is: Does this test fail without this change?

Yes.